tooling/scripts/test-categories.yaml

```diff
@@ -32,6 +32,7 @@ group-01:
   - kafka
   - kafka-sasl
   - mllp
+  - milvus
```

Comment:
You can remove this since we're just adding as JVM mode only.

Reply:
Done